orcid-message-listener/src/main/resources/orcid-message-listener-web-context.xml

```diff
@@ -21,36 +21,28 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:context="http://www.springframework.org/schema/context"
 	xmlns:cache="http://www.springframework.org/schema/cache" xmlns:aop="http://www.springframework.org/schema/aop"
 	xmlns:jms="http://www.springframework.org/schema/jms"
+	xmlns:util="http://www.springframework.org/schema/util"
 	xsi:schemaLocation="
         http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.1.xsd
         http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.1.xsd
         http://www.springframework.org/schema/cache http://www.springframework.org/schema/cache/spring-cache.xsd
         http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-3.1.xsd 
-        http://www.springframework.org/schema/jms http://www.springframework.org/schema/jms/spring-jms-4.1.xsd">
-
-	<!--  
-	<context:property-placeholder
-		location="classpath:message-listener.properties"
-		ignore-resource-not-found="true" ignore-unresolvable="true" /> -->
-		
-    <!-- ${org.orcid.persistence.properties} gets replaced during the Maven
-       build, but other properties are escaped from Maven build replacement by preceding
-       with \ and so are handled by Spring. -->
-    <context:property-placeholder location="\${org.orcid.message-listener.config.file:}" ignore-resource-not-found="true" ignore-unresolvable="true" />
-    <context:property-placeholder location="classpath:${org.orcid.message-listener.properties}" ignore-resource-not-found="true"
-        ignore-unresolvable="true" />
+        http://www.springframework.org/schema/jms http://www.springframework.org/schema/jms/spring-jms-4.1.xsd
+        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util-2.0.xsd">
+	
+    <context:property-placeholder location="${org.orcid.message-listener.properties}" />
```

Comment:
@TomDemeranville  In order to deploy this to the servers, we need to be able to configure the properties file path, so, now on, in order to start the server locally, we will need a new property: 

-Dorg.orcid.message-listener.properties=classpath:message-listener.properties